### PR TITLE
Mark RecoverVolumeExpansionFailure as beta

### PR DESCRIPTION
**Pull request by gnufied**
xref https://github.com/kubernetes/enhancements/issues/1790

### CODE REVIEW

File: content/en/docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -481,8 +481,8 @@ administrator intervention.
 {{% feature-state for_k8s_version="v1.23" state="alpha" %}}
```
Comment (sftim) [inline content/en/docs/concepts/storage/persistent-volumes.md:481]: You should change this line as well
```suggestion
{{< feature-state feature_gate_name="RecoverVolumeExpansionFailure" >}}
```

**Review by sftim — COMMENTED** [review]
Thanks for the PR.

Please read https://kubernetes.io/docs/contribute/new-content/new-features/#ready-for-review-feature-gates to learn about the extra changes you are likely to need to make.

/hold
This change should target the _dev-1.32_ branch; _main_ is the live website.

File: content/en/docs/concepts/storage/persistent-volumes.md (hunk 1 above)
Comment (gnufied) [reply content/en/docs/concepts/storage/persistent-volumes.md:481]: Yes, I am going to update this and rest of the stuff.

**Review by dipesh-rawat — COMMENTED** [review]
@gnufied The PR is currently lacking the change to transition the feature gate `RecoverVolumeExpansionFailure` from Alpha to Beta. The necessary change should be made in the feature gate description file - 
`recover-volume-expansion-failure.md` ([here](https://github.com/kubernetes/website/blob/dev-1.31/content/en/docs/reference/command-line-tools-reference/feature-gates/recover-volume-expansion-failure.md?plain=1)).

Please refer to the documentation [here](https://kubernetes.io/docs/contribute/new-content/new-features/#ready-for-review-feature-gates) for further information.

**Comment by rdalbuquerque** [comment]
Hello @gnufied  :wave:! I'm reaching out from the Docs team. Just checking in as we approach Docs Freeze on Tuesday November 26th 18:00 PDT. This documentation appears to still be under review. To meet the Docs Freeze, this PR must have a technical review as well as lgtm and approve labels applied, without any unaddressed comments or concerns from SIG Docs. Thank you!

**Pull request merged**